### ISSUE TITLE
Form field has aria-invalid attribute when error is false

### ISSUE THREAD

**Issue by aoelen** [Semantic-Org/Semantic-UI-React#4042]
### Steps
Use a form field, for example:
`
<Form.Field
    id='form-input-control-error-email'
      control={Input}
      label='Email'
      placeholder='joe@schmoe.com'
      error={false}
    />
`

### Expected Result
The aria-invalid attribute should not be present (or should be false)

### Actual Result
The aria-invalid attribute is set to true:

`<input aria-describedby="form-input-control-error-email-error-message" aria-invalid="true" placeholder="joe@schmoe.com" id="form-input-control-error-email" type="text">`

### Version
1.2.0

**Comment by layershifter** [comment]
Thanks for reporting. It seems that condition there should be simpler 🐱 


`src/collections/Form/FormField.js`
```diff
    'aria-describedby': ariaDescribedBy,
-   'aria-invalid': error !== undefined ? true : undefined,
+   'aria-invalid': error ? true : undefined,
```

Would you like to submit a PR?

**Comment by aoelen** [comment]
Done, I added some tests as well

**Comment by layershifter** [comment]
A fix was released in `semantic-ui-react@1.2.1` 🎉
